templates/bamboo/0/rancher-compose.yml

```diff
@@ -0,0 +1,77 @@
+.catalog:
+  name: "Bamboo"
+  version: "v0.0.1"
+  description: "Bambo Server"
+  uuid: bamboo-0
+  minimum_rancher_version: v0.51.0
+  questions:
+    - variable: bamboo_version
+      description: "Version of Bamboo Server. Check https://www.atlassian.com/software/bamboo/download?os=linux for current version."
+      label: "Bamboo Version"
+      required: true
+      default: "5.14.1"
+      type: "string"
+    - variable: http_port
+      description: "HTTP port to access Bamboo Server UI"
+      label: "Bamboo HTTP Port"
+      required: true
+      default: "8085"
+      type: "int"
+    - variable: agent_port
+      description: "HTTP port to connect Bamboo Agents. Can be useful when connecting agents out of Rancher."
+      label: "Agent HTTP Port"
+      required: true
+      default: "54663"
+      type: "int"
```

Comment:
http_port and agent_port questions don't needed if not mapping host ports directly.